appcues/src/main/java/com/appcues/data/mapper/experience/ExperienceMapper.kt

```diff
@@ -150,11 +151,10 @@ internal class ExperienceMapper(
     }
 
     private fun List<ExperienceTrait>.getExperiencePresentingTraitOrThrow(): PresentingTrait {
-        return filterIsInstance<PresentingTrait>().firstOrNull() ?: throw AppcuesPresentingTraitNotFound()
+        return filterIsInstance<PresentingTrait>().firstOrNull()
+            ?: throw AppcuesTraitException("Presenting capability trait required")
```

Comment:
getting rid of `AppcuesPresentingTraitNotFound` and consolidating as just another type of `AppcuesTraitException` here.